dhall/tests/Dhall/Test/Import.hs

```diff
@@ -129,27 +136,32 @@ successTest path = do
 
                                 return m
 
-                        let reset Nothing = do
+                            reset Nothing = do
```

Comment:
Please keep the `let` to stick with the existing code style.

Reply:
Oops, didn't mean to remove this thanks